elfi/examples/gnk.py

```diff
@@ -0,0 +1,223 @@
+    (2009): 189-201.
+
+    Parameters
+    ----------
+    A : float
+        The location.
+    B : float
+        The scale.
+    g : float
+        The skewness.
+    k : float
+        The kurtosis.
+    c : float, optional
+        The overall asymmetry parameter, as a convention fixed to 0.8 [2].
+    n_obs : int, optional
+        The number of the observed points
+    batch_size : int, optional
+    random_state : np.random.RandomState, optional
+
+    Returns
+    -------
+    array_like
+        The yielded points.
+    """
+
+    # Standardising the parameters
+    A = np.asanyarray(A).reshape((-1, 1))
+    B = np.asanyarray(B).reshape((-1, 1))
+    g = np.asanyarray(g).reshape((-1, 1))
+    k = np.asanyarray(k).reshape((-1, 1))
+
+    # Sampling from the z term, Equation 1, [2].
+    z = ss.norm.rvs(size=(batch_size, n_obs), random_state=random_state)
+
+    # Yielding Equation 1, [2].
+    term_exp = (1 - np.exp(-g * z)) / (1 + np.exp(-g * z))
+    y = A + B * (1 + c * (term_exp)) * (1 + z**2)**k * z
+
+    return y
+
+
+def get_model(n_obs=50, true_params=None, stats_summary=None, seed_obs=None):
+    """Returns an initialised univariate g-and-k model.
+
+    Parameters
+    ----------
+    n_obs : int, optional
+        The number of the observed points.
+    true_params : array_like, optional
+        The parameters defining the model.
+    stats_summary : array_like, optional
+        The chosen summary statistics, expressed as a list of strings.
+    seed_obs : np.random.RandomState, optional
+
+    Returns
+    -------
+    elfi.ElfiModel
+    """
+    m = elfi.ElfiModel(set_current=False)
+
+    # Initialising the default parameter settings as given in [2].
+    if true_params is None:
+        true_params = [3, 1, 2, .5]
+    if stats_summary is None:
+        # stats_summary = ['ss_order']
+        stats_summary = ['ss_robust']
+        # stats_summary = ['ss_octile']
+
+    # Initialising the default prior settings as given in [2].
+    elfi.Prior('uniform', 0, 10, model=m, name='A')
+    elfi.Prior('uniform', 0, 10, model=m, name='B')
+    elfi.Prior('uniform', 0, 10, model=m, name='g')
+    elfi.Prior('uniform', 0, 10, model=m, name='k')
+
+    y_obs = GNK(*true_params, n_obs=n_obs,
+                random_state=np.random.RandomState(seed_obs))
+
+    fn_sim = partial(GNK, n_obs=n_obs)
+    elfi.Simulator(fn_sim, m['A'], m['B'], m['g'], m['k'], observed=y_obs,
+                   name='GNK')
+
+    # Initialising the chosen summary statistics.
+    fns_summary_all = [ss_order, ss_robust, ss_octile]
+    fns_summary_chosen = []
+    for fn_summary in fns_summary_all:
+        if fn_summary.__name__ in stats_summary:
+            summary = elfi.Summary(fn_summary, m['GNK'],
+                                   name=fn_summary.__name__)
+            fns_summary_chosen.append(summary)
+
+    elfi.Distance('euclidean', *fns_summary_chosen, name='d')
+
+    return m
+
+
+def ss_order(y):
```

Comment:
It seems like these functions are duplicated in `gnk.py` and `bignk.py`. It's probably better to put them in the same file and import the models in the `examples` package with the `__init__.py` file.